### DISCUSSION TITLE
I can't display my list with PnP - Search Results

### DISCUSSION THREAD

**Discussion by Netamine** [microsoft-search/pnp-modern-search#2575] [Q&A]
Hi Guys,
I'm trying to display my SharePoint list with PnP - Search Results but the only column working is Title

<img width="484" alt="image" src="https://user-images.githubusercontent.com/118911273/203572107-4546eb09-50dd-4ba5-a53b-4ec284f57a0d.png">

**Comment by wobba** [comment]
Have you mapped crawl properties and reindexed items after the mapping?

**Reply by Netamine** [reply]
Hello,
yes i did but unfortunately without success.
However i did something else and it works ! I think it's a question of synchronization.
what i did was to create a new properties  linked to mapped crawled properties before creating my list.

<img width="634" alt="image" src="https://user-images.githubusercontent.com/118911273/203799984-abc68fb3-c175-48a7-b472-580ca2d764d2.png">

once I have created my imported list from an Excel file,
on my sharepoint page when I created the PnP object - Search Results it worked on the first try
Thank you  for your interaction.
